Sources/AsyncHTTPClient/ConnectionPool.swift

```diff
@@ -368,11 +370,57 @@ final class ConnectionPool {
             }
         }
 
+        private func makeNonTSBootstrap(on eventLoop: EventLoop) throws -> NIOClientTCPBootstrap {
```

Comment:
I would recommend moving it back to `Utils`

Reply:
Maybe we should move the whole flow of creating a NIOClientTCPBootstrap to Utils. Doesn't make sense we have one half in ConnectionPool.swift and the other in Utils

Reply:
👍

Reply:
done